test/test_store_base_sti_class.rb

```diff
@@ -155,13 +155,15 @@ def test_join_association
     assert SpecialTag.joins(:polytaggings).where(id: tag.id).first
   end
 
-  def test_where_query
-    tag = SpecialTag.create!(:name => 'Special')
-    post = @thinking_post
+  if Gem::Version.new(ActiveRecord::VERSION::STRING) >= Gem::Version.new('5')
```

Comment:
Shouldn't this test be applied for all versions?

Reply:
Glad you're back :)

I can't really remember exactly why and how I didn't add tests for rails 4 but I guess it might be because the fix was breaking compatibility for lower versions.
By looking back at the code I guess I found a way to fix latest version but didn't have time to fix older ones, and hoped that since it had not bothered anyone by that time it would be ok.

Reply:
I'll merge this as is, and file a bug for older versions. Thanks!